sdk/src/types/api/plugins/participation/responses.rs

```diff
@@ -60,3 +60,23 @@ pub struct AddressOutputsResponse {
     /// Outputs is a map of output status per output_id.
     pub outputs: HashMap<OutputId, OutputStatusResponse>,
 }
+
+#[cfg(test)]
+impl OutputStatusResponse {
```

Comment:
Not a big fan of these tbh, can't we move them to the test crate directly? Or to a dedicated module like we have for the block module?